pushmi_pullyu.gemspec

```diff
@@ -24,9 +24,13 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.3.1'
 
   spec.add_runtime_dependency 'activesupport', '~> 5.0'
+  spec.add_runtime_dependency 'connection_pool', '~> 2.2'
   spec.add_runtime_dependency 'daemons', '~> 1.2', '>= 1.2.4'
   spec.add_runtime_dependency 'redis', '~> 3.3'
-  spec.add_runtime_dependency 'connection_pool', '~> 2.2'
+  spec.add_runtime_dependency 'rollbar', '~> 2.14'
+
+  # Rollbar recommended json serializer & version
+  spec.add_runtime_dependency 'oj', '~> 2.12.14'
```

Comment:
Old docs? OJ gem is on version 3.0.8 now: 
https://github.com/ohler55/oj
https://rubygems.org/gems/oj

And looks like it follows Sem Ver too